pkg/sdn/plugin/controller.go

```diff
@@ -336,10 +339,49 @@ func (plugin *OsdnNode) SetupSDN(localSubnetCIDR, clusterNetworkCIDR, servicesNe
 	return true, nil
 }
 
+func (plugin *OsdnNode) updateEgressNetworkPolicyFailureLabel(failure bool) error {
+	node, err := plugin.registry.kClient.Nodes().Get(plugin.hostName)
+	if err != nil {
+		return fmt.Errorf("could not find Node record for %q: %v", plugin.hostName, err)
+	}
+	if failure {
+		if node.Labels == nil {
+			node.Labels = make(map[string]string)
+		}
+		node.Labels[EgressNetworkPolicyFailureLabel] = "true"
+	} else {
+		label, ok := node.Labels[EgressNetworkPolicyFailureLabel]
+		if !ok || label != "true" {
+			return nil
+		}
+		delete(node.Labels, EgressNetworkPolicyFailureLabel)
+	}
+
+	_, err = plugin.registry.kClient.Nodes().UpdateStatus(node)
+	if err != nil {
+		return fmt.Errorf("could not update Node: %v", err)
```

Comment:
Just return the error directly.  We don't want to mask it and this isn't providing extra information.